javascript/src/main/components/Nav/AppNavbar.js

```diff
@@ -8,73 +8,78 @@ import useSWR from "swr";
+        </NavDropdown>
           {isAdmin && (
-            <NavDropdown title="Admin">
-              <NavDropdown.Item as={Link} to={"/admin/panel"}>
-                Admin Panel
-              </NavDropdown.Item>
-              <NavDropdown.Item as={Link} to={"/admin/settings"}>
-                Admin Settings
-              </NavDropdown.Item>
-            </NavDropdown>
+              <NavDropdown title="Admin">
+                  <NavDropdown.Item as={Link} to={"/admin/panel"}>
+                      Admin Panel
+                  </NavDropdown.Item>
+                  <NavDropdown.Item as={Link} to={"/admin/settings"}>
+                      Admin Settings
+                  </NavDropdown.Item>
+              </NavDropdown>
           )}
-          <LinkContainer to={"/about"}>
+        <LinkContainer to={"/about"}>
             <Nav.Link>About</Nav.Link>
-          </LinkContainer>
-          <NavDropdown title="Statistics">
-            <NavDropdown.Item as={Link} to="/statistics/numFullCoursesByDept">
-              Full Classes by Department
-            </NavDropdown.Item>
-            <NavDropdown.Item as={Link} to="/statistics/courseOccupancy">
-              Course Occupancy by Department
-            </NavDropdown.Item>
-            <NavDropdown.Item
-              as={Link}
-              to="/statistics/courseOccupancyByDivision"
-            >
-              Course Occupancy by Class Division
-            </NavDropdown.Item>
-            <NavDropdown.Item as={Link} to="/statistics/classSize">
-              Average Class Size by Department
-            </NavDropdown.Item>
-            <NavDropdown.Item as={Link} to="/statistics/totalCourses">
-              Total Courses by Department
-            </NavDropdown.Item>
-            <NavDropdown.Item as={Link} to="/statistics/numOpenCoursesByDept">
-              Open Courses by Department
-            </NavDropdown.Item>
-          </NavDropdown>
-          {(isAdmin || isMember) && <ScheduleNav />}
-        </Nav>
-        <Navbar.Collapse className="justify-content-end">
-          <AuthNav />
-        </Navbar.Collapse>
+        </LinkContainer>
+        <NavDropdown title="Statistics">
+          <NavDropdown.Item as={Link} to="/statistics/numFullCoursesByDept">
+            Full Classes by Department
+          </NavDropdown.Item>
+          <NavDropdown.Item as={Link} to="/statistics/courseOccupancy">
+            Course Occupancy by Department
+          </NavDropdown.Item>
+          <NavDropdown.Item as={Link} to="/statistics/courseOccupancyByDivision">
+            Course Occupancy by Class Division
+          </NavDropdown.Item>
+          <NavDropdown.Item as={Link} to="/statistics/classSize">
+            Average Class Size by Department
+          </NavDropdown.Item>
+          <NavDropdown.Item as={Link} to="/statistics/fullDeptSummary">
+            Full Classes Summary
+          </NavDropdown.Item>
```

Comment:
Weird... idk how this got here, but this is the link to the page Matt and I started work on. It's just a blank page atm with no functionality. It probably needs to go.

Reply:
Commit 7dd4226 addresses this.